packages/automerge-repo/src/network/NetworkAdapter.ts

```diff
@@ -11,27 +11,25 @@ import { StorageId } from "../storage/types.js"
  * until the adapter emits a `ready` event before it starts trying to use it
  */
 export abstract class NetworkAdapter extends EventEmitter<NetworkAdapterEvents> {
-  peerId?: PeerId // hmmm, maybe not
```

Comment:
The "hmm, maybe not" here is that the network adapter gets its peerId at connect() time but we have it on the constructor. Perhaps we should clean that up?

Reply:
? It's not on the constructor - the abstract class doesn't specify a constructor interface, and none of the implementations take a peerId in the constructor, only in `connect()`.

Reply:
yes -- of course, my misread there. thanks.